community/kernel/src/test/java/org/neo4j/kernel/impl/store/NeoStoresTest.java

```diff
@@ -756,9 +774,13 @@ public void shouldCloseAllTheStoreEvenIfExceptionsAreThrown() throws Exception
         {
             // then
             assertEquals( "Cannot stop in state: rotating", ex.getMessage() );
-            // and the page cache closes with no errors
-            pageCache.close();
         }
+
+        thread.interrupt();
+        thread.join();
```

Comment:
It'd just be nice if you could assert that only the InterruptedException was thrown, and not any other exception.